packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/DownloadListenerHostApiImpl.java

```diff
@@ -5,40 +5,63 @@
 package io.flutter.plugins.webviewflutter;
 
 import android.webkit.DownloadListener;
-import io.flutter.plugins.webviewflutter.GeneratedAndroidWebView.DownloadListenerFlutterApi;
 
 class DownloadListenerHostApiImpl implements GeneratedAndroidWebView.DownloadListenerHostApi {
   private final InstanceManager instanceManager;
   private final DownloadListenerCreator downloadListenerCreator;
-  private final GeneratedAndroidWebView.DownloadListenerFlutterApi downloadListenerFlutterApi;
+  private final DownloadListenerFlutterApiImpl flutterApi;
+
+  static class DownloadListenerImpl implements DownloadListener, Releasable {
+    private final Long instanceId;
+    private final DownloadListenerFlutterApiImpl flutterApi;
+    private boolean ignoreCallbacks = false;
```

Comment:
I think this would benefit from a comment explaining what it's for, so people don't have to go read the implementation to understand it.